README.md

```diff
@@ -112,6 +112,25 @@ To have the CI working, you must:
 - Link your repository with [Coveralls](https://coveralls.io/repos/new).
 - (Optional) Insert your [Nx Cloud](https://nx.app/) access token in github secrets under `NX_CLOUD_TOKEN`. This enables for caching and faster build times.
 
+### Deployment
```

Comment:
This section should be at the very end of the file as it is the last thing you want to do.

Reply:
Its at the end of Usage 🦈